src/sp_config.c

```diff
@@ -61,6 +62,26 @@ int parse_empty(char *restrict line, char *restrict keyword, void *retval) {
   return 0;
 }
 
+int parse_list(char *restrict line, char *restrict keyword, void *list_ptr) {
```

Comment:
`keyword` can be a `const char *restrict` I think

Reply:
Doing so add compilation warning.